README.md

```diff
@@ -68,7 +68,7 @@ DESCRIPTION
   This command has three modes: ruleset, diff-only, and directory.
      Ruleset mode (default) compares two files and applies a ruleset to determine if any changes are breaking.
      Diff-only mode compares two files to determine if there are any differences, without applying a ruleset.
-     Directory mode compares all the files in two directories and determines if there are any differences.
+     Directory mode finds all exchange.json files in two directories recursively and compares all the spec files described in them. Applies the default ruleset.
```

Comment:
How about adding a sample directory structure to make it clear in one of the docs. May be in library-usage?

Reply:
done